gslides_api/element/table.py

```
@@ -738,13 +821,69 @@ def _calculate_proportional_heights_after_addition(
+        border_positions = ["TOP", "BOTTOM", "INNER_HORIZONTAL"]
+        return [
+            UpdateTableBorderPropertiesRequest(
+                objectId=self.objectId,
+                tableRange=TableRange(
+                    location=TableCellLocation(rowIndex=0, columnIndex=0),
+                    rowSpan=self.table.rows,
+                    columnSpan=self.table.columns,
+                ),
+                borderPosition=position,
+                tableBorderProperties=TableBorderProperties(
+                    weight=Dimension(magnitude=weight_emu, unit=Unit.EMU)
```

Ruff (0.14.11)

Undefined name `Unit`

(F821)